packages/cli/src/commands/hydrogen/deploy.ts

```diff
@@ -99,6 +99,12 @@ export default class Deploy extends Command {
       required: false,
       default: false,
     }),
+    'auth-bypass-token-duration': Flags.string({
+      description:
+        'Specify the duration (in hours) up to 12 hours for the authentication bypass token. Defaults to `2`',
+      required: false,
+      dependsOn: ['auth-bypass-token'],
```

Comment:
It might be useful to add an `env` flag name to both auth-bypass-token and auth-bypass-token-duration so that devs can pass these values from env variables.